lms/static/scripts/frontend_apps/components/GroupConfigSelector.tsx

```diff
@@ -233,12 +231,8 @@ export default function GroupConfigSelector({
 
   return (
     <>
-      <LabeledCheckbox
+      <Checkbox
         checked={useGroupSet}
-        id={checkboxId}
-        // The `name` prop is required by LabeledCheckbox but is unimportant
-        // as this field is not actually part of the submitted form.
-        name="use_group_set"
```

Comment:
These props are no longer required in the new `Checkbox` component, so removing them as they were not used with any particular purpose other than fulfilling the contract requirement.